playbooks/spark-aarch64-build/run.yaml

```diff
@@ -0,0 +1,84 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build spark
+      shell:
+        cmd: |
+          set -ex
+          apt-get update -y
+
+          # Install java
+          apt-get install default-jre -y
+          apt-get install default-jdk -y
+          java_home=$(dirname $(dirname $(update-alternatives --list javac)))
+          echo "export JAVA_HOME=${java_home}" >> ~/.profile
+          echo "export PATH=${java_home}/bin:$PATH" >> ~/.profile
+          source ~/.profile
+
+          # Install maven
+          wget http://www.us.apache.org/dist/maven/maven-3/3.6.1/binaries/apache-maven-3.6.1-bin.tar.gz
+          tar -xvf apache-maven-3.6.1-bin.tar.gz
+          export PATH=$PWD/apache-maven-3.6.1/bin:$PATH
+
+          # Install vim
```

Comment:
This is a demo to show out current Spark testing fails on aarch64, so I think we should run upstream spark code directly and show the fails in testing log, then apply our workaround and test again, that will prove our solution worked for Spark community.

Reply:
If we run test spark code directly then run test apply with the workaround, maybe it will be too long?

Reply:
Spark UT will failed in 5 minutes without workaround after compiling stage, then Maven will apply local repoistory package and don't download again, so it will not waste much time.

Reply:
OK, thanks, I will try it first.

Reply:
I don't see the comment have been resolved in latest patch, so please don't mark it as resovled.